src/platform/web/lib/util/webstorage.ts

```diff
@@ -1,4 +1,3 @@
-import * as Utils from 'common/lib/util/utils';
```

Comment:
If the intention was that this PR be reviewed commit by commit (which, given the "See commit messages for more details" instruction it presumably was), then 84a99908ad1e3b38c5c0acec6be4adacf0a727a2 is just kinda noise for reviewers — shouldn’t this change just go into the commit that introduced the linter error? Ditto 3d41166d5a7d6a0cbf8b361e9571a9b2f62c5b03.

Reply:
> If the intention was that this PR be reviewed commit by commit

I didn't intend it from the beginning, but given the type of changes done in this PR it totally makes sense to go commit by commit instead of looking at everything at once.

I agree it's a noise, will squash those.

Reply:
Squashed